File: cpp/Leetcode/c++/problem645.cpp
#include<bits/stdc++.h>
using namespace std;

//beats 5% 1517ms
class Solution {
public:
    vector<int> findErrorNums(vector<int>& nums) {
        int dup=0,missing=0;
        int n=nums.size();
        vector<int>res;
        for(int i=1;i<n+1;i++){
            int cnt=count(nums.begin(),nums.end(),i);
            if(cnt==2){
                dup=i;
            }else if(cnt==0){
                missing=i;
            }
        }
        res.push_back(dup);
        res.push_back(missing);
        return res;   
    }
};

File: cpp/Leetcode/Python/problem645.py
#beats 5% 8280ms
class Solution:
    def findErrorNums(self, nums):
        n=len(nums)
        dup=0
        missing=0
        for i in range(1,n+1):
            count=nums.count(i)
            if(count==2):
                dup=i
            elif(count==0):
                missing=i
        return [dup,missing]